service/test/agama/storage/manager_test.rb

```diff
@@ -378,13 +380,16 @@
     let(:devicegraph) { "staging-plain-partitions.yaml" }
 
     it "copy needed files, installs the bootloader, sets up the snapshots, " \
-       "copy logs, runs the post-installation scripts, and umounts the file systems" do
+       "copy logs, symlink resolv.conf, runs the post-installation scripts, " \
+       "unlink resolv.conf and umounts the file systems" do
```

Comment:
NP


```suggestion
       "unlink resolv.conf, and umounts the file systems" do
```

Reply:
Sorry, I maybe was wrong, not sure. I though the documentation SUSE guide styles recommended to use the Oxford comma, but now I have doubts :sweat_smile: 

https://documentation.suse.com/style/current/html/docu_styleguide/sec-language.html#sec-comma

> Use commas to separate elements in a series of three or more elements, but do not put a comma before the conjunction in most simple series. For example, “Find basic information about how to register your system, modules and extensions.” Use commas around phrases like for example and that is. Introductory phrases at the beginning of a sentence are normally followed by a comma. For example, “Before using YaST Online Update, configure a network connection.”

Although the key is to know if your sentence is considered a "simple serie".

Reply:
> Although the key is to know if your sentence is considered a "simple serie".

I'd say no, it isn't.